bin/blazegraph_load_api.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# On a debian package installation, the original file could be found at:
+#/usr/bin/loadRestAPI.sh
+
+if [ "$1" == "" -o "$1" == "-h" -o "$1" == "--help" ]; then
+    echo "Usage: $0 {file_or_directory} [{graph_namespace}]"
+    exit
+else
+    FILE_OR_DIR=$1
+fi
+
+if [ ! -z "$2" ]; then
+    NAMESPACE=$2
+else
+    NAMESPACE=kb
+fi
+
+[ -f /etc/default/blazegraph ] && source /etc/default/blazegraph
+[ -z "$JETTY_PORT" ] && JETTY_PORT=9999
```

Comment:
The config file with the details for the deployment to blazegraph-dev is in
https://github.com/sul-dlss/shared_configs/blob/ld4p-rdf-to-blazegraph-dev/config/config_blazegraph.sh